### PR TITLE
pcap_dnsproxy 0.4.5

### PR DESCRIPTION
**Pull request by blackgear**

### CODE REVIEW

File: Library/Formula/pcap_dnsproxy.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class PcapDnsproxy < Formula
   desc "Powerful DNS proxy designed to anti DNS spoofing"
   homepage "https://github.com/chengr28/Pcap_DNSProxy"
-  url "https://github.com/chengr28/Pcap_DNSProxy/archive/v0.4.4.tar.gz"
-  sha256 "4412e5f6b8bd1cf985f69132a1a38fcf3bf81ba83e9439920ae3237cf059816b"
+  url "https://github.com/chengr28/Pcap_DNSProxy/archive/v0.4.5.tar.gz"
+  sha256 "22861dba30b90d43b8623b71e5d02300968f1ef9f4fc341a6a0ce6dd315e8004"
   head "https://github.com/chengr28/Pcap_DNSProxy.git"
   revision 1
```
Comment (DomT4) [inline Library/Formula/pcap_dnsproxy.rb]: Please remove `revision` lines on stable version updates, Thanks!

File: Library/Formula/pcap_dnsproxy.rb [hunk 2]
```diff
@@ -13,6 +13,7 @@ class PcapDnsproxy < Formula
     sha256 "33c919db4973f2fbb44f0e35a5d2c381744622f67b4dfe4bc97423b2b0a9c184" => :mavericks
   end
 
+  depends_on :macos => :yosemite
```
Comment (DomT4) [inline Library/Formula/pcap_dnsproxy.rb]: Why?

**Comment by dunn** [comment]
Failing on 10.9: http://bot.brew.sh/job/Homebrew%20Pull%20Requests/40116/version=mavericks/testReport/junit/brew-test-bot/mavericks/install_pcap_dnsproxy/
```
Undefined symbols for architecture x86_64:
  "_pcap_close", referenced from:
      CaptureModule(pcap_if const*, bool) in Capture.o
  "_pcap_compile", referenced from:
      CaptureModule(pcap_if const*, bool) in Capture.o
  "_pcap_datalink", referenced from:
      CaptureModule(pcap_if const*, bool) in Capture.o
  "_pcap_findalldevs", referenced from:
      CaptureInit() in Capture.o
  "_pcap_freealldevs", referenced from:
      CaptureInit() in Capture.o
  "_pcap_freecode", referenced from:
      CaptureModule(pcap_if const*, bool) in Capture.o
  "_pcap_geterr", referenced from:
      CaptureModule(pcap_if const*, bool) in Capture.o
  "_pcap_lib_version", referenced from:
      ReadCommand(int, char**) in Main.o
  "_pcap_loop", referenced from:
      CaptureModule(pcap_if const*, bool) in Capture.o
  "_pcap_open_live", referenced from:
      CaptureModule(pcap_if const*, bool) in Capture.o
  "_pcap_setfilter", referenced from:
      CaptureModule(pcap_if const*, bool) in Capture.o
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

**Comment by DomT4** [comment]
Ah, was the >Mavericks OS X dep here added originally because this project now expects to build against a certain Xcode version?

> 升级项目文件到 Xcode 7.1 (Mac)

**Comment by mikemcquaid** [comment]
> Ah, was the >Mavericks OS X dep here added originally because this project now expects to build against a certain Xcode version?

Sounds like it. :+1: to adding back.

**Comment by blackgear** [comment]
Xcode requirements added back

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/4a87d88c737b33af6f250284fbdd389d9f2395f7. Thank you for another contribution to Homebrew @blackgear! :smiley_cat:

**Pull request closed without merging**